docs/project/faq.md

```diff
@@ -182,6 +197,62 @@ If a migration occurs, we anticipate:
+[Carbon Language Discord server](https://discord.com/channels/655572317891461132/996113200464543804).
+
+GitHub Issues should be reserved for missing features, bugs, and anything else
+that is fixable via a Pull Request.
+
+### What are the prerequisites for contributing to Carbon Language's design and tools??
+
+**[Carbon Language isn't ready for use](#how-soon-can-we-use-carbon). This
+section is for people wishing to participate in designing and implementing the
+language.**
+
+Carbon is being designed to migrate C++ codebases and to look familiar to
+C++ programmers. As such, familiarity with C++ is very important. Carbon is also
+trying to learn from other programming languages, so having broad experience
+with other programming languages will be helpful to see tradeoffs in design
+decisions.
+
+The Carbon toolchain is being implemented in C++, and we also use Python and
+Starlark. As we're building off of the LLVM project, familiarity with Clang and
+other parts of LLVM will be advantageous, but not required.
+
+Our [contribution tools](/docs/project/contribution_tools.md) page documents
+specific tools we use when building.
+
+### Why aren't people willing to discuss certain topics, like the file extension or syntax for specific constructs?
+
+When it comes to designing a language, there are lots of decisions where there
+are many workable options, and all of them will get the job done without serious
+issues. We need to pick _one_ in order to make progress, and no choice is going
+to appeal to everyone's preferences. We try to base the decision on consistent
+rationale, but different people will weigh factors differently.
+
+Importantly: these decisions have a fairly small impact on the overall language
+design. If we had written prototype code with `.car` instead of `.carbon`, the
+core language design would still proceed. What does matter is that we _pick_ one
+of the options, and try to do so with some rationale and consistency.
+
+It can be tempting to revisit these, especially when there's community
+disagreement. However, because the language is still very incomplete, it is
+important that we focus our time and energy on the open questions. This
+is one reason why we're often a bit reluctant to revisit some of the more minor
+decisions. It's not that they're _definitely_ right, or that they are truly set
+in stone. It's just a prioritization of making progress on the open questions
+because at least we have an answer and that answer is probably good enough for
+now. The signal to really revisit stuff is new information. There was an example
+with digit separators: we had never considered how many _domain_ conventions we
+would want to support and so had overly restricted where the separators go.
+
+We encourage folks to either shrug and let quirky syntax be quirky, or look for
+the objective use case that was clearly missed in the previous decision and
+makes the current state _sharply_ problematic before pushing for change.
```

Comment:
I feel like we should say something along the lines of:

When there is disagreement between options, comments saying "I like X" or "I don't like X" don't really contribute to the conversation. Much more helpful would be a reason for or against X that hasn't already been considered.

Reply:
Done.